### PR TITLE
[doc] Add basics fix typo in use-cases

### PR DESCRIPTION
**Pull request by SKuerten**
Add basics to the online documentation

### CODE REVIEW

File: doc/basics.rst [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
+
+.. code:: ipython3
+
+    display(m_mu.name())
+    display(m_mu.latex())
+    display(m_mu.evaluate())
+
+
+
+.. parsed-literal::
```
Comment (dvandyk) [inline doc/basics.rst:195]: Uhh, I did not know about this!

File: doc/basics.rst [hunk 2]
```diff
@@ -0,0 +1,753 @@
...
+pertain to a single process. Consequently, naming of kinematic variables
+does not require a distinction by any sort of ``prefix``. We will see
+what kinematic variables a process requires in a later section. You can
+create an empty set of kinematic variables as follows:
+
+.. code:: ipython3
+
+    kinematics = eos.Kinematics()
+
+We can populate this object with a choice of kinematic variables: - We
```
Comment (dvandyk) [inline doc/basics.rst:313]: Itemization not working!

File: doc/basics.rst [hunk 3]
```diff
@@ -0,0 +1,753 @@
...
+variables. This makes it possible to investigate correlations among
+observables that share a kinematic variable (e.g., LFU ratios such as
+:math:`R_K` as a functions of the lower dilepton momentum cut-off).
+
+****************************
+Options and what they impact
+****************************
+
+EOS allows us to modify the behaviour of processes through objects of
+the ``eos.Options`` class. In many cases, the processes have a default
```
Comment (dvandyk) [inline doc/basics.rst:444]: ``eos.Option``, not ``eos.Options``.

File: doc/basics.rst [hunk 4]
```diff
@@ -0,0 +1,753 @@
...
+set of options, e.g., the process’s default choice of hadronic form
+factors, its default choice of BSM model, and so on. In some cases, it
+does not make sense to have a default choice, e.g., when evaluating a
+single hadronic form factor. In such cases, you will see an error
+expressed through a Python exception.
```
Comment (dvandyk) [inline doc/basics.rst:449]: ``[...] if the mandatory option is not specified by you!``

File: doc/basics.rst [hunk 5]
```diff
@@ -0,0 +1,753 @@
...
+You can create a new and empty set of options as follows:
+
+.. code:: ipython3
+
+    options = eos.Options()
+
+We can now populate this object with individual options. Options are
+pairs of strings. Within each pair, we refer to the first element as the
+**key** and to the second element as the **value**. Typical keys
+include: - ``model``, to select a BSM model; - ``form-factors``, to
```
Comment (dvandyk) [inline doc/basics.rst:466]: itemization!

File: doc/basics.rst [hunk 6]
```diff
@@ -0,0 +1,753 @@
...
+*******************************************************
+
+EOS uses the class ``eos.Observable`` to provide theory predictions for
+a variety of processes and their associated observables. A list of all
+observables known to EOS is available online in the documentation, or
+via:
+
+.. code:: ipython3
+
+     # The list is rather long; uncomment the following line by removing the leading '#'
```
Comment (dvandyk) [inline doc/basics.rst:571]: Please remove the comment and the leading '#', since this is not a notebook.

File: doc/basics.rst [hunk 7]
```diff
@@ -0,0 +1,753 @@
...
+    <table>
+    <tr><th><tt style="color:grey">l</tt></th><td>mu</td></tr>
+    <tr><th><tt style="color:grey">model</tt></th><td>WilsonScan</td></tr>
+    <tr><th><tt style="color:grey">q</tt></th><td>s</td></tr>
+    </table>
+
+
+
+
+Any observable can be handled just like any other Python object. For
```
Comment (dvandyk) [inline doc/basics.rst:718]: ``Any`` or ``An``?

File: doc/use-cases.rst [hunk 8]
```diff
@@ -876,4 +876,4 @@ The results look as follows:
    :width: 600
 
 .. image:: /images/use-cases_simulation_hist2d-0-3.png
-   :width: 600
\ No newline at end of file
+   :width: 600
```
Comment (dvandyk) [inline doc/use-cases.rst:879]: Please undo this change, whatever it is :smile:

**Review by dvandyk — CHANGES_REQUESTED** [review]

**Comment by SKuerten** [comment]
I'm not completely done yet. I messed up some rst syntax. Will fix that later

File: doc/basics.rst [hunk 9]
```diff
@@ -0,0 +1,753 @@
...
+parameters, from which your favourite parameter can be extracted and
+looked at.
+
+.. code:: ipython3
+
+    parameters = eos.Parameters()
+
+The variable ``parameters`` now contains all of your parameters. You can
+use ``display(parameters)`` to see the full list, which is rather
+lengthy. Let’s have a look at something more manageable: the parameters
```
[the remark this answers is not in the document]
Comment (SKuerten) [reply doc/basics.rst:44]: I think that's a misunderstanding. parameters is defined in line 40 and contains the complete lists of parameters, thus the plural. I don't think singular makes sense here?

[the remark this answers is not in the document]
Comment (dvandyk) [reply doc/basics.rst:44]: But there is only a single parameter for the muon mass!

**Review by dvandyk — APPROVED** [review]

**Pull request merged**